forte_wrapper/twitter/twittersearch_processor.py

```diff
@@ -84,15 +86,16 @@ def _process(self, input_pack: MultiPack):
         r"""Search using Twitter API to fetch tweets for a query.
         This query should be contained in the input multipack with name
         `self.config.query_pack_name`.
-        This method adds new packs to `input_pack` containing the retrieved
-        results. Each result is added as a `ft.onto.base_ontology.Document`.
+        Each result is added as a new data pack, and a
+        ft.onto.base_ontology.Document annotation is used to cover the whole
```

Comment:
Use back tips around code-like stuff.  i.e. `ft.onto.base_ontology.Document` instead of ft.onto.base_ontology.Document